app/config/index.js

```diff
@@ -5,15 +5,18 @@ const { LucidLog } = require('lucid-log');
 let logger;
 
 function getConfigFile(configPath) {
-	try {
-		return require(path.join(configPath, 'config.json'));
-	} catch (e) {
-		return null;
-	}
+	return require(path.join(configPath, 'config.json'));
 }
 
 function argv(configPath, appVersion) {
-	let configFile = getConfigFile(configPath);
+	configPath = "/home/kwe/dev_private/teams-for-linux/"
```

Comment:
this can't be your path as it will not work for other people.

Reply:
Whoops, that was a debug leftover.
Will remove it when I'm back on m6u pc 😅